doc/build-osx.md

```diff
@@ -60,7 +60,7 @@ brew install llvm
 And append the following to the configure commands below:
```

Comment:
Perhaps this phrase should be changed as well since there is no `configure` command anymore?

Reply:
The word "confgure" is still used in all the build docs to describe that step (not just mac), so I'll leave that as is. Maybe another PR can reword the descriptions if we want to do that.